backend/cfg_selectgen.ml

```diff
@@ -131,6 +131,40 @@ let next_instr_id () : int =
   incr next_instr_id;
   res
 
+(* XXX mshinwell: move the next two functions into Cfg proper? *)
+let make_instr desc arg res dbg =
+  { Cfg.desc;
+    arg;
+    res;
+    dbg;
+    fdo = Fdo_info.none;
+    live = Reg.Set.empty;
+    stack_offset = -1;
+    id = next_instr_id ();
+    irc_work_list = Unknown_list;
+    ls_order = 0;
+    available_before =
+      Some (Reg_availability_set.Ok Reg_with_debug_info.Set.empty);
```

Comment:
@mshinwell is that definitely ok? (I can't
remember why it is not simply `None`.)

Reply:
Not sure offhand, will leave a CR for when we come to debugging work.